### DISCUSSION TITLE
pcl2 用的是什么字体？

### DISCUSSION THREAD

**Discussion by 212112111zwl** [Meloong-Git/PCL#962] [提问] [answered]
就想知道一下

**Comment by WorldHim** [comment]
微软雅黑（？

**Comment by WForst-Breeze** [comment, marked as answer]
（个人猜测和辨别的，具体的我也不太清楚）
中文：应该直接就是微软雅黑，这个我不太确定
英文和数字：就是此仓库中 resources/font.ttf 中的字体（PCL English）

**Reply by WForst-Breeze** [reply]
某些字体识别网站识别中文字体是方正兰亭黑简体，不过这个字体商用不免费，我觉得不大可能，况且此仓库里也没找到任何有关方正兰亭黑简体的文件qwq

**Reply by 212112111zwl** [reply]
谢谢~

**Comment by zengmaoyuan** [comment]
在Application.Xaml中有`<Setter Property="FontFamily" Value="Resources/#PCL English, Microsoft YaHei UI" />`，说明英文为PCL English, 中文为 Microsoft YaHei UI

**Comment by SqrtTan** [comment]
话说PCL这类“半开源软件”能免费用雅黑吗（

**Reply by CiiLu** [reply]
> 话说PCL这类“半开源软件”能免费用雅黑吗（

> Windows comes with a collection of fonts that are installed as system-wide resources. Any application installed on your Windows computer has access to these fonts.
A Windows application can use the fonts to render content to a screen, allow that content to be edited, and allow that content to be output to a device, like a printer. （https://learn.microsoft.com/en-us/typography/fonts/font-faq ）


在 Windows 中软件使用系统内置字体显示是不需要授权的
